Ziti.NET.Standard/src/OpenZiti/Native/nAPI.cs

```diff
@@ -345,9 +345,13 @@ public struct ziti_options {
         //public IntPtr config_types;
         public IntPtr /*public char**/ config_types;
 
-        public Int32 api_page_size;
-
+        public Int32 api_page_size;
```

Comment:
this should just be changed to 'int' or 'uint'. i'm shocked i didn't hit this issue

Reply:
done